logback-ecs-encoder/src/main/java/co/elastic/logging/logback/EcsEncoder.java

```diff
@@ -114,7 +119,11 @@ public void close() throws IOException {
         }
         IThrowableProxy throwableProxy = event.getThrowableProxy();
         if (throwableProxy instanceof ThrowableProxy) {
```

Comment:
Also apply the configured throwableConverter if throwableProxy is not an instance of `ThrowableProxy`

```java
        IThrowableProxy throwableProxy = event.getThrowableProxy();
        if (throwableProxy != null) {
            if (throwableConverter != null) {
                EcsJsonSerializer.serializeException(builder, throwableProxy.getClassName(), throwableProxy.getMessage(), throwableConverter.convert(event), stackTraceAsArray);
            } else if (throwableProxy instanceof ThrowableProxy) {
                EcsJsonSerializer.serializeException(builder, ((ThrowableProxy) throwableProxy).getThrowable(), stackTraceAsArray);
            } else {
                EcsJsonSerializer.serializeException(builder, throwableProxy.getClassName(), throwableProxy.getMessage(), throwableProxyConverter.convert(event), stackTraceAsArray);
            }
        }
```

Reply:
updated. sorry for the delay